src/functions.js

```diff
@@ -626,19 +626,117 @@ defineFunction([
 defineFunction([
     "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
     "\\check", "\\hat", "\\vec", "\\dot",
-    // We don't support expanding accents yet
-    // "\\widetilde", "\\widehat"
+    "\\widehat", "\\widetilde", "\\overrightarrow", "\\overleftarrow",
+    "\\Overrightarrow", "\\overleftrightarrow", "\\overgroup",
+    "\\overlinesegment", "\\overleftharpoon", "\\overrightharpoon",
 ], {
     numArgs: 1,
 }, function(context, args) {
     const base = args[0];
+
+    const isStretchy = !utils.contains([
+        "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+        "\\check", "\\hat", "\\vec", "\\dot",
+    ], context.funcName);
+
+    const isShifty = !isStretchy || utils.contains([
+        "\\widehat", "\\widetilde", "\\overrightarrow", "\\overrightharpoon",
+    ], context.funcName);
```

Comment:
It's time to talk about arrow accents. Below are four images of the exact same function.
![four-examples](https://cloud.githubusercontent.com/assets/16403058/26418102/dea6ba2c-406f-11e7-8dd1-ac84b4db3505.PNG)
That's from LaTeX, AMS-LaTeX,  the mathabax package, and the MnSymbol package. Why are they all so different? Well, I have a guess. The following is from AMS-LaTeX:
![four-accents](https://cloud.githubusercontent.com/assets/16403058/26418439/0e269c08-4071-11e7-8b6e-38be7fe577ef.PNG)
One of those accents is not like the others. That is, the `\overrightarrow` is much taller than the others. AMS fixed the collision that was in plain LaTeX.  But their glyph is still much taller than other accents. I think mathabax and MnSymbol are sized more like accents.

So I used an arrowhead from KaTeX-AMS font. It's 0.334 em high instead of 0.522 em high. For comparison, mine is in the middle of this list:
![five-examples](https://cloud.githubusercontent.com/assets/16403058/26419063/4ce7133a-4073-11e7-92b3-37218d391051.PNG)
So now we have a choice to make. Naturally, I think my arrow is the best choice. MathJax chose to follow the AMS-LaTeX accent. I can support either my arrow or the AMS-LaTeX one. I would like to avoid the other three.

Let me know what you decide.